File: templates/Uwp/_comp/MTM/Service.IdentityForcedLogin._VB/ViewModels/LogInViewModel_postaction.vb
'{[{
Imports Microsoft.Toolkit.Mvvm.ComponentModel
Imports Microsoft.Toolkit.Mvvm.Input
'}]}
Namespace ViewModels
    Public Class LogInViewModel

        Public Property IsBusy As Boolean
            Get
                Return _isBusy
            End Get
            Set(value As Boolean)
                [SetProperty](_isBusy, value)
'{[{
                LoginCommand.NotifyCanExecuteChanged()
'}]}
            End Set
        End Property
    End Class
End Namespace

File: templates/Uwp/_comp/MVVMToolkit/Service.IdentityForcedLogin/ViewModels/LogInViewModel_postaction.cs
//{[{
using Microsoft.Toolkit.Mvvm.ComponentModel;
using Microsoft.Toolkit.Mvvm.Input;
//}]}

namespace Param_RootNamespace.ViewModels
{
    public class LogInViewModel : ObservableObject
    {
        public bool IsBusy
        {
            get => _isBusy;
            set
            {
                SetProperty(ref _isBusy, value);
//{[{
                LoginCommand.NotifyCanExecuteChanged();
//}]}
            }
        }
    }
}
